docs/recommendations/general.md

```diff
@@ -64,7 +64,7 @@ Make sure you have also seen the [Basics](../background/basics.md), explaining t
 - HGVS recommends following the [HGNC guidelines](https://www.genenames.org/about/guidelines/) and the [VICC Gene Fusion Specification](https://fusions.cancervariants.org/en/latest) nomenclature to describe products of gene fusions.
     - The HGNC recommendations include using a `GENESYMBOL1::GENESYMBOL2` syntax for gene-level fusion descriptions, and a `GENESYMBOL1-GENESYMBOL2` syntax for read-through transcripts.
     - The VICC nomenclature extends the HGNC recommendations to include a terminology, information model, and nomenclature for gene-level and exon-level representation, with components for disambiguating regulatory fusions from chimeric transcript fusions.
-    - HGVS also recommends the use of [adjoined transcripts](RNA/splicing.md) for precise and unambiguous characterization of chimeric transcripts at the sequence level.
+    - HGVS also recommends the use of [adjoined transcripts](RNA/adjoined_transcript.md) for precise and unambiguous characterization of chimeric transcripts at the sequence level.
```

Comment:
This change prompted me to check the splicing page because I was wondering why we linked to that page to explain adjoined transcripts. I see that the splicing page still lists an example of an adjoined transcript and links to SVD-WG007. Maybe we should remove that? I don't see why the splicing page should have that example, nor why somebody looking for adjoined transcripts would check the splicing page :thinking: